core/modules/layout/layout.module

```diff
@@ -1812,6 +1812,23 @@ function layout_context_required_by_path($path) {
   return $required_contexts[$path];
 }
 
+/**
+ * Return the current user context, which is always available.
+ */
+function layout_current_user_context() {
+  global $user;
```

Comment:
You don't need this line since you're accessing the `$GLOBALS` directly further down.

Reply:
ah, yes. using GLOBALS is what the original code did. is that preferred?